lab/components/HtmlFragment.vue

```diff
@@ -0,0 +1,30 @@
+<script>
+/* eslint-disable no-underscore-dangle */
+// https://stackoverflow.com/a/47332729/2766908
+import Vue from 'vue/dist/vue';
+
+// https://stackoverflow.com/a/67771171/2766908
+// https://jsfiddle.net/Linusborg/mfqjk5hm/
+/**
+ * unlike "v-html" this renders html content
+ * without a wrapping container
+ */
+export default {
+	functional: true,
+	props: {
+		html: {
+			type: String,
+			required: true,
+		},
+	},
+	render(h, context) {
+		const nodes = new Vue({
+			beforeCreate() {
+				this.$createElement = h;
+			},
+			template: `<div>${context.props.html}</div>`,
+		}).$mount()._vnode.children;
+		return nodes;
+	},
+};
+</script>
```

Comment:
Would rather have you use [vue-frag](https://github.com/privatenumber/vue-frag) here since it's a battle-tested solution.

Reply:
I don't know what I'm doing wrong but I tried using vue-frag and couldn't make it work... I tried:
```
<fragment v-html="svgText" />
```
and
```
<div v-frag v-html="svgText" />
```
and neither worked. No errors during build. No errors in the console at run-time. Just doesn't render. No clue why.

Reply:
Sorry for the late response.

So, you can't set `v-html` on a Fragment because it doesn't exist in the DOM (same way a real DOM Fragment doesn't exist in the DOM and [doesn't support `innerHTML`](https://stackoverflow.com/questions/8202195/using-document-createdocumentfragment-and-innerhtml-to-manipulate-a-dom)).

I noticed `svgText` just returns a single SVG element though. Is a fragment necessary?